### ISSUE TITLE
Mekanism Crash Issue (PLEASE HELP)

### ISSUE THREAD

**Issue by Digg1** [aidancbrady/Mekanism#2740]
So I downloaded the mod for minecraft 1.7.10, and it worked fine. The problem occurs in a quite strange way. Take any machine of choice (for me it was the crusher), now to power that machine you have to connect it to any type of universal cable. But whenever I try to connect it to the cable, the game crashes and force quits, giving this message. I used a basic universal cable on a crusher btw. If you can help, please do so.

---- Minecraft Crash Report ----
// I let you down. Sorry :(

Time: 13.06.2015 00:38
Description: Ticking memory connection

java.util.NoSuchElementException: None.get
	at scala.None$.get(Option.scala:322)
	at scala.None$.get(Option.scala:320)
	at codechicken.multipart.MultiPartRegistry$.writePartID(MultiPartRegistry.scala:191)
	at codechicken.multipart.TileMultipart.writeAddPart(TileMultipart.scala:289)
	at codechicken.multipart.TileMultipart.addPart_impl(TileMultipart.scala:275)
	at codechicken.multipart.MultipartGenerator$.addPart(MultipartGenerator.scala:97)
	at codechicken.multipart.TileMultipart$.addPart(TileMultipart.scala:601)
	at codechicken.multipart.TItemMultiPart$class.place$1(ItemMultiPart.scala:37)
	at codechicken.multipart.TItemMultiPart$class.onItemUse(ItemMultiPart.scala:47)
	at codechicken.multipart.JItemMultiPart.func_77648_a(ItemMultiPart.scala:14)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at scala.None$.get(Option.scala:322)
	at scala.None$.get(Option.scala:320)
	at codechicken.multipart.MultiPartRegistry$.writePartID(MultiPartRegistry.scala:191)
	at codechicken.multipart.TileMultipart.writeAddPart(TileMultipart.scala:289)
	at codechicken.multipart.TileMultipart.addPart_impl(TileMultipart.scala:275)
	at codechicken.multipart.MultipartGenerator$.addPart(MultipartGenerator.scala:97)
	at codechicken.multipart.TileMultipart$.addPart(TileMultipart.scala:601)
	at codechicken.multipart.TItemMultiPart$class.place$1(ItemMultiPart.scala:37)
	at codechicken.multipart.TItemMultiPart$class.onItemUse(ItemMultiPart.scala:47)
	at codechicken.multipart.JItemMultiPart.func_77648_a(ItemMultiPart.scala:14)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)

-- Ticking connection --
Details:
	Connection: net.minecraft.network.NetworkManager@6aadf20f
Stacktrace:
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 203852800 bytes (194 MB) / 405479424 bytes (386 MB) up to 2134114304 bytes (2035 MB)
	JVM Flags: 7 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M -Xmx2048m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP v9.05 FML v7.10.85.1291 Minecraft Forge 10.13.2.1291 9 mods loaded, 9 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	FML{7.10.85.1291} [Forge Mod Loader] (forge-1.7.10-10.13.2.1291.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Forge{10.13.2.1291} [Minecraft Forge] (forge-1.7.10-10.13.2.1291.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ForgeMultipart{1.1.2.331} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Mekanism{8.1.4} [Mekanism] (Mekanism-1.7.10-8.1.4.232.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MekanismGenerators{8.1.4} [MekanismGenerators] (MekanismGenerators-1.7.10-8.1.4.232.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MekanismTools{8.1.4} [MekanismTools] (MekanismTools-1.7.10-8.1.4.232.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	McMultipart{1.1.2.331} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ForgeMicroblock{1.1.2.331} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['DiggerBill'/290, l='New World', x=182,92, y=80,74, z=248,24]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by aidancbrady** [comment]
Please report this to FMP's issue tracker, this doesn't appear to be related to Mekanism.  Let me know if I'm wrong!

**Comment by Digg1** [comment]
Hey Aidan so I've been trying several methods (such as downgrading Forge, use older versions of Minecraft, use older versions of Java) to solve this odd bug. I have also missed a detail, which was that the game crashed whether the universal cable was connecting to anything or not. I couldn't really find the FMP issue tracker as well. I'll include my latest crash report here so you can see if anything's changed (different pc). Help will be greatly appreciated! (The forge version might've changed as well) P.S: (If it's really hard for you to figure out, please include a link to the issue tracker :D) (PPS: I've researched other issues like mine, so no I don't use optifine)

Crash Log:

---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 12.07.2015 20:28
Description: Ticking memory connection

java.util.NoSuchElementException: None.get
	at scala.None$.get(Option.scala:322)
	at scala.None$.get(Option.scala:320)
	at codechicken.multipart.MultiPartRegistry$.writePartID(MultiPartRegistry.scala:191)
	at codechicken.multipart.TileMultipart.writeAddPart(TileMultipart.scala:289)
	at codechicken.multipart.TileMultipart.addPart_impl(TileMultipart.scala:275)
	at codechicken.multipart.MultipartGenerator$.addPart(MultipartGenerator.scala:97)
	at codechicken.multipart.TileMultipart$.addPart(TileMultipart.scala:601)
	at codechicken.multipart.TItemMultiPart$class.place$1(ItemMultiPart.scala:37)
	at codechicken.multipart.TItemMultiPart$class.onItemUse(ItemMultiPart.scala:47)
	at codechicken.multipart.JItemMultiPart.func_77648_a(ItemMultiPart.scala:14)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at scala.None$.get(Option.scala:322)
	at scala.None$.get(Option.scala:320)
	at codechicken.multipart.MultiPartRegistry$.writePartID(MultiPartRegistry.scala:191)
	at codechicken.multipart.TileMultipart.writeAddPart(TileMultipart.scala:289)
	at codechicken.multipart.TileMultipart.addPart_impl(TileMultipart.scala:275)
	at codechicken.multipart.MultipartGenerator$.addPart(MultipartGenerator.scala:97)
	at codechicken.multipart.TileMultipart$.addPart(TileMultipart.scala:601)
	at codechicken.multipart.TItemMultiPart$class.place$1(ItemMultiPart.scala:37)
	at codechicken.multipart.TItemMultiPart$class.onItemUse(ItemMultiPart.scala:47)
	at codechicken.multipart.JItemMultiPart.func_77648_a(ItemMultiPart.scala:14)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)

-- Ticking connection --
Details:
	Connection: net.minecraft.network.NetworkManager@3faad29c
Stacktrace:
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 105301496 bytes (100 MB) / 354185216 bytes (337 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1490 9 mods loaded, 9 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAAAAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAAAAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1490-1.7.10.jar) 
	UCHIJAAAAAAAAA	Forge{10.13.4.1490} [Minecraft Forge] (forge-1.7.10-10.13.4.1490-1.7.10.jar) 
	UCHIJAAAAAAAAA	ForgeMultipart{1.1.2.331} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	UCHIJAAAAAAAAA	Mekanism{8.1.5} [Mekanism] (Mekanism-1.7.10-8.1.5.245.jar) 
	UCHIJAAAAAAAAA	MekanismGenerators{8.1.5} [MekanismGenerators] (MekanismGenerators-1.7.10-8.1.5.245.jar) 
	UCHIJAAAAAAAAA	MekanismTools{8.1.5} [MekanismTools] (MekanismTools-1.7.10-8.1.5.245.jar) 
	UCHIJAAAAAAAAA	McMultipart{1.1.2.331} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	UCHIJAAAAAAAAA	ForgeMicroblock{1.1.2.331} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['DiggerBill'/67, l='New World', x=-697,96, y=4,00, z=76,29]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

thank you for your time :)

**Comment by RAWRwins254** [comment]
@Digg1 Forge multipart (FMP) requires CodeChickenLib which is a dependency of CodeChickenCore and can only be obtained via installing CCC. This is listed on the readme section for the FMP repo here on github. Here's the Link: https://github.com/Chicken-Bones/ForgeMultipart

**Comment by Digg1** [comment]
@RAWRwins254 @aidancbrady Well folks, I've downloaded the CodeChickenLib, and it didn't really help (the game crashed with the placement of a universal cable). After that, I've downloaded CCC and other CodeChicken stuff. I can see CodeChicken Core, ChickenChunks, Forge Multipart (the 3 Mekanism files) , Minecraft Multipart Plugin and Forge Microblocks in my mod list. I presume it has something to do with other than Forge stuff. Anyone can still help if they can.

**Comment by RAWRwins254** [comment]
@Digg1 about the best thing I can suggest is update FMP to 1.2.0.345. You're currently on .331. I'll look into mekanism's code (build.gradle or whichever build file it's located in if at all) and post back.

**Comment by RAWRwins254** [comment]
So after doing some digging in the build scripts, .331 is the correct one to use at minimum. Hmmmm. @aidancbrady Could it really be FMP considering he's put in CCC and CCL per requests?

@Digg1 Can you post your most recent crash report? I want to make sure of matches between mods since adding ChickenChunks, CCC, and CCL.

**Comment by Digg1** [comment]
@RAWRwins254  I increased the max heap size to 4 gigs and then to 12, but it showed no difference. I have the crash log here:

---- Minecraft Crash Report ----
// On the bright side, I bought you a teddy bear!

Time: 12.07.2015 16:09
Description: Ticking memory connection

java.util.NoSuchElementException: None.get
	at scala.None$.get(Option.scala:322)
	at scala.None$.get(Option.scala:320)
	at codechicken.multipart.MultiPartRegistry$.writePartID(MultiPartRegistry.scala:191)
	at codechicken.multipart.TileMultipart.writeAddPart(TileMultipart.scala:289)
	at codechicken.multipart.TileMultipart.addPart_impl(TileMultipart.scala:275)
	at codechicken.multipart.MultipartGenerator$.addPart(MultipartGenerator.scala:97)
	at codechicken.multipart.TileMultipart$.addPart(TileMultipart.scala:601)
	at codechicken.multipart.TItemMultiPart$class.place$1(ItemMultiPart.scala:37)
	at codechicken.multipart.TItemMultiPart$class.onItemUse(ItemMultiPart.scala:47)
	at codechicken.multipart.JItemMultiPart.func_77648_a(ItemMultiPart.scala:14)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at scala.None$.get(Option.scala:322)
	at scala.None$.get(Option.scala:320)
	at codechicken.multipart.MultiPartRegistry$.writePartID(MultiPartRegistry.scala:191)
	at codechicken.multipart.TileMultipart.writeAddPart(TileMultipart.scala:289)
	at codechicken.multipart.TileMultipart.addPart_impl(TileMultipart.scala:275)
	at codechicken.multipart.MultipartGenerator$.addPart(MultipartGenerator.scala:97)
	at codechicken.multipart.TileMultipart$.addPart(TileMultipart.scala:601)
	at codechicken.multipart.TItemMultiPart$class.place$1(ItemMultiPart.scala:37)
	at codechicken.multipart.TItemMultiPart$class.onItemUse(ItemMultiPart.scala:47)
	at codechicken.multipart.JItemMultiPart.func_77648_a(ItemMultiPart.scala:14)
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505)
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)

-- Ticking connection --
Details:
	Connection: net.minecraft.network.NetworkManager@92a3496
Stacktrace:
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 4027008024 bytes (3840 MB) / 4281597952 bytes (4083 MB) up to 4281597952 bytes (4083 MB)
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms1G -Xmx4G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M -Xms4096M -Xmx4096M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1448 11 mods loaded, 11 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1448-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1448} [Minecraft Forge] (forge-1.7.10-10.13.4.1448-1.7.10.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.7.46} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	ChickenChunks{1.3.4.19} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.19-universal.jar) 
	UCHIJAAAA	ForgeMultipart{1.1.2.331} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	UCHIJAAAA	Mekanism{8.1.5} [Mekanism] (Mekanism-1.7.10-8.1.5.246.jar) 
	UCHIJAAAA	MekanismGenerators{8.1.5} [MekanismGenerators] (MekanismGenerators-1.7.10-8.1.5.246.jar) 
	UCHIJAAAA	MekanismTools{8.1.5} [MekanismTools] (MekanismTools-1.7.10-8.1.5.246.jar) 
	UCHIJAAAA	McMultipart{1.1.2.331} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	UCHIJAAAA	ForgeMicroblock{1.1.2.331} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['DiggerBill'/2, l='New World', x=737,69, y=4,00, z=-103,26]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by RAWRwins254** [comment]
@Digg1 hmmmm... last i knew CCC was dependent on a matching CCL of which those two versions don't match. CCC version as listed in the development branch's build.properties file is 1.0.6.39. @aidancbrady might be able to shed more light on this than me though. But until he replies, try using CCC 1.0.6.39 instead of 1.0.7.46 and see if that fixes the problem.

**Comment by Digg1** [comment]
@RAWRwins254 @aidancbrady In my time of absence, I tried many things such as downgrading/upgrading java, forge and the mod version. I also tried using CCC 1.0.6.39 as a solution. None of those seem to work. However, during my experimenting, I've seen that universal cables were not the only sources of crashing. I've seen that Mechanical Pipes, Pressurized Tubes and Logistical Transporters also cause the game to crash. While these were a problem, I've seen that Restrictive transporters and diversion transporters don't cause any problem. If you want their crash logs, I'll include them later.
